solaris/solaris11/generate_wazuh_packages.sh

```diff
@@ -99,12 +101,20 @@ compile() {
     arch="$(uname -p)"
     # Build the binaries
     if [ "$arch" = "sparc" ]; then
-        gmake -j $THREADS TARGET=agent PREFIX=${install_path} USE_SELINUX=no USE_BIG_ENDIAN=yes
+        if [ "${disa_share}" == "true" ]; then
+            gmake -j $THREADS TARGET=agent DISABLE_SHARED=yes PREFIX=${install_path} USE_SELINUX=no USE_BIG_ENDIAN=yes || exit 1
+        else
+            gmake -j $THREADS TARGET=agent PREFIX=${install_path} USE_SELINUX=no USE_BIG_ENDIAN=yes || exit 1
+        fi
     else
-        gmake -j $THREADS TARGET=agent PREFIX=${install_path} USE_SELINUX=no
+        if [ "${disa_share}" == "true" ]; then
+            gmake -j $THREADS TARGET=agent DISABLE_SHARED=yes PREFIX=${install_path} USE_SELINUX=no || exit 1
+        else
+            gmake -j $THREADS TARGET=agent PREFIX=${install_path} USE_SELINUX=no || exit 1
+        fi      
     fi
 
-    DISABLE_SHARED=yes .$SOURCE/install.sh
+    $SOURCE/install.sh || exit 1
```

Comment:
If the script hasn't built the `libwazuhext.so` before, the install.sh will build it because that's the default option. To avoid this, you must run the script as follows: `DISABLE_SHARED=${disa_share} .$SOURCE/install.sh || exit 1`.

Reply:
Added

Reply:
Hello,

I had to correct a point that had escaped in line install.sh in https://github.com/wazuh/wazuh-packages/pull/283/commits/fe581832e89693533ebbcf618e082ec4fde78d10